### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by sanieni6**
## Milestone 1
Changes done to the portfolio project:
+ Add  the section for works
+ Add the new  about me section

### CODE REVIEW

**Comment by sanieni6** [comment]
## Milestone 1
Changes done:
+ Solve problems with linters

File: index.html [hunk 1]
```diff
@@ -37,13 +40,150 @@ <h1 class="supporting-text">I'm a software developer</h1>
...
+  </section>
+  <section class="about-me-section">
+    <div class="about-me">
+      <h3 class="about-title">About me</h3>
+      <p class="description-about">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </div>
+    <hr class="solidhr1">
+    <div class="skills">
+      <div class="about-me-sectiones">
```
Comment (Tadesse-Alemayehu) [inline index.html:152]: - [ ] Your design is impressive @sanieni6, But there is a minor detail missing for this section, As you can see from the screenshot your design could use more spacing between contents and your decoration image can pop-out to the view a bit.
### Your design 
Screenshot 2023-01-14 081735

### Figma design
Screenshot 2023-01-14 081832

File: index.html [hunk 2]
```diff
@@ -37,13 +40,150 @@ <h1 class="supporting-text">I'm a software developer</h1>
...
+    </section>
 
-    </ul>
+  </section>
+  <section class="about-me-section">
+    <div class="about-me">
+      <h3 class="about-title">About me</h3>
+      <p class="description-about">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </div>
```
Comment (Tadesse-Alemayehu) [inline index.html:149]: - [ ] Your button is really great, But we need to follow the button styling: `Active`, `Hover`, and `Pressed`, Please add these effects for a better user interface.
Screenshot 2023-01-14 081956

File: index.html [hunk 3]
```diff
@@ -37,13 +40,150 @@ <h1 class="supporting-text">I'm a software developer</h1>
...
+          <li class="des-text">Javascript</li>
+          <li class="des-text">HTML</li>
+          <li class="des-text">CSS</li>
+        </ul>
+      </div>
+      <div class="about-me-sectiones">
+        <div class="holder-image">
+          <img src="images/rectangle.png" alt="">
+          <h3 class="format">Frameworks</h3>
+        </div>
```
Comment (Tadesse-Alemayehu) [inline index.html:167]: - [ ] Its awesome that you make technology in each category(e.g., "JavasScript", "HTML" and "CSS") a `ul `with each being inside `li`, But we also need to follow the same structure for technology categories ("Languages", "Frameworks" and "Skills"). Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

**Review by Tadesse-Alemayehu — CHANGES_REQUESTED** [review]
### Hi @sanieni6 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## HighligAshley
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sanieni6** [comment]
## Apply required changes
Changes done:
+ Add more spacing between contents About me section
+ Apply button styling
+ Use an unordened list for skills and technologies

**Review by IngDavidTM — APPROVED** [review]
Hi @sanieni6 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**